### ISSUE TITLE
DeprecationWarnings when running versioneer install

### ISSUE THREAD

**Issue by hf-krechan** [python-versioneer/python-versioneer#228]
Hello together,

I am currently trying to build a good development environment for a new Python package.
During my research I came across versioneer and was instantly hooked.
But when running versioneer install I got the following DeprecationWarnings:

```shell
(dev) PS E:\workspaces\BO4E-python> versioneer install
versioneer.py (0.18) installed into local tree
Now running 'versioneer.py setup' to install the generated files..
versioneer.py:342: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
  parser = configparser.SafeConfigParser()
versioneer.py:344: DeprecationWarning: This method will be removed in future versions.  Use 'parser.read_file()' instead.
  parser.readfp(f)
```

Information about my environment:
* Python 3.8
* OS: Windows 10 Pro, Version 2004

I am still learning how to use git in a proper way and I am not so sure how to create a good pull request.
So sorry for the inconvenience.

**Comment by effigies** [comment]
This has been addressed in #207. The next release, which should be coming out in the near future, will not emit these warnings.

**Comment by hf-krechan** [comment]
Nice, thanks for the quick feedback =)

**Comment by effigies** [comment]
Fixed in 0.19.
